tokens/base.json

```
@@ -522,75 +522,75 @@
     },
     "toasted-marshmallow": {
       "100": {
-        "value": "#fdf7e7",
+        "value": "#fdf6e6",
         "type": "color"
       },
       "200": {
-        "value": "#ebd6a8",
+        "value": "#ebd6a9",
         "type": "color"
       },
       "300": {
-        "value": "#e6bf6b",
+        "value": "#e6bf6c",
         "type": "color"
       },
       "400": {
-        "value": "#cd9f3c",
+        "value": "#CC9E3B",
         "type": "color"
       },
       "500": {
-        "value": "#b27f10",
+        "value": "#b37f10",
         "type": "color"
       },
       "600": {
-        "value": "#8a5e00",
+        "value": "#8C6000",
         "type": "color"
       }
     },
     "coconut": {
       "100": {
-        "value": "#f1efef",
+        "value": "#F0EEEE",
         "type": "color"
       },
       "200": {
-        "value": "#e3dede",
+        "value": "#e3dfdf",
         "type": "color"
       },
       "300": {
-        "value": "#d1cccd",
+        "value": "#d1cbcc",
         "type": "color"
       },
       "400": {
-        "value": "#b3adad",
+        "value": "#b3acac",
         "type": "color"
       },
       "500": {
-        "value": "#9d9595",
+        "value": "#9e9595",
         "type": "color"
       },
       "600": {
-        "value": "#8e8586",
+        "value": "#8F8687",
         "type": "color"
       }
     },
     "cappucino": {
```

Let's go ahead and fix this spelling now. ☕
```suggestion
    "cappuccino": {
```